### ISSUE TITLE
Missing sourcecode to the sample XBE

### ISSUE THREAD

**Issue by robhshu** [Gabriel-Maldonado/XboxHLE#1]
What is the expected behaviour of the 'nonxdk samples'?

**Comment by maldonadogabriel** [comment]
Sure thing, I will include the source samples soon. But for now drawpixel, you can find that and several other samples at xbdev.net.

**Comment by robhshu** [comment]
Thanks for the hint. I found the source for the setpixel demo here: http://www.xbdev.net/non_xdk/openxdk/setpixel/setpixel.php

On a side note I'm aiming to update OpenXDK to support a bit more functionality here: https://github.com/robhshu/OpenXDK

I'm also interested to see what you do with XboxHLE :smile:

**Comment by maldonadogabriel** [comment]
Sorry this is late, but I've included the source for both samples in the repository.

I used the drawPixel sample found here http://www.xbdev.net/non_xdk/nasm_xbe/xbe_040/index.php. I'm unsure of the differences in the Setpixel demo, but I do think it needs to be compiled using OpenXDK or maybe even MSXDK.

I'm very interested in OpenXDK and would like to get a few samples running on XboxHLE at a later date, but first I need to figure out on how to get the samples provided from xbdev to compile, the last time I tried I was unsuccessful. If you happen to know of a guide or can provide me some helpful steps, that would be most appreciated.

Thanks! I'm excited that your interested, I can only hope that this will be something meaningful to everyone in the future. I'm aware of the hurdles present in Xbox emulation, even so this is something I think I can continue to improve.
